### PR TITLE
Add README, tasks, moved to pnpm and added an ignore for react

### PR DESCRIPTION
**Pull request by TrainDoctor**
Updates template to latest library version as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```bash
+sudo npm i -g pnpm
+```
+
+### Installation
+
+1. Clone the repository to use as an example for making your plugin.
+2. In your clone of the repository run these commands:
+   1. ``pnpm i``
+   2.  (Optional, needed to solve build errors)
```
Comment (AAGaming00) [inline README.md:24]: This is misleading, IMO should just be under some useful commands/notes section for how to update the plugin library version.

**Review by AAGaming00 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TrainDoctor) [reply README.md:23]: > This is misleading, IMO should just be under some useful commands/notes section for how to update the plugin library version.

Addressed in latest commit, changed from "Installation" to "Keeping Your Plugin Updated" and dropped usage of "clone" etc.

Comment (AAGaming00) [reply README.md:23]: Did not mean change installation, meant to move the part about updating the lib to somewhere separate. Installation should really be "Getting started" or something like that.

Comment (TrainDoctor) [reply README.md:23]: > Did not mean change installation, meant to move the part about updating the lib to somewhere separate. Installation should really be "Getting started" or something like that.

Ah gotcha. The suggested change thing is always a couple lines up for you so if you want mobile to stop janking you out probably best to go a few lines down from what you want changed. 👍🏻

**Review by AAGaming00 — APPROVED** [review]
LGTM

**Pull request merged**